index.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express")
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+//Route to get a stock from the database. Calling https://codubee-projects-api.herokuapp.com/stocks/getStockData
+app.get('/getStockData', function(postmanReq, postmanRes) {
+    console.log("Getting Data for a Stock...")
+    axios.get('https://codubee-projects-api.herokuapp.com/stocks/getStockData')
+    .then((response) => {
+        console.log(response.data)
+        postmanRes.status(200).json({"message":response.data})

```

Comment:
Here the key is message. Would a key of message be good, or is a key of data better?

Reply:
Agree with @Codubee